vecgeom-fix-for-arm64.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b3014e9..f10fae0 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -317,6 +317,8 @@ elseif(Intel)
+       list(APPEND CMAKE_CXX_FLAGS -std=c++11)
+     elseif(CMAKE_CXX_STANDARD STREQUAL 14)
+       list(APPEND CMAKE_CXX_FLAGS -std=c++14)
++    elseif(CMAKE_CXX_STANDARD STREQUAL 17)
++      list(APPEND CMAKE_CXX_FLAGS -std=c++17)
+     else()
+       message(FATAL_ERROR "Unsupported C++ standard requested")
+     endif()
+diff --git a/base/Global.h b/base/Global.h
+index 9856628..d90b1f8 100644
+--- a/base/Global.h
++++ b/base/Global.h
+@@ -31,7 +31,18 @@ using uint = unsigned int;
+ #endif
+ #else
+ // Functionality of <mm_malloc.h> is automatically included in icc
++#if defined(__i386__) || defined(__x86_64__)
+ #include <mm_malloc.h>
++#else
++#if __cplusplus >= 201703L
++#include <cstdlib>
++#define _mm_malloc(a, b) ::aligned_alloc(b, a)
```

Comment:
Just a comment. I couldn't use here std:: namespace due to a bug in GCC. It's easy to fix and there is a ticket for GCC 8, but apparently it could break target in Windows thus not being fixed at this moment.